molecule/scenario_resources/tests/test_default.rb

```diff
@@ -8,3 +8,28 @@
     it { should be_enabled }
   end
 end
+
+'pulpcore-webserver' do |webserver|
+  describe port(80) do
+    it { should be_listening }
+  end
+
+  describe port(443) do
+    it { should be_listening }
+  end
+
+  describe http('http://localhost/pulp/api/v3/status',
+                 ssl_verify: false, max_redirects: 1) do
+      its('status') { should eq 200 }
+      its('body') { should match /database_connection/ }
+    end
+  end
+
+  describe http('https://localhost/pulp/api/v3/status',
+                 ssl_verify: false, max_redirects: 1) do
+      its('status') { should eq 200 }
+      its('body') { should match /database_connection/ }
+    end
+  end
```

Comment:
Loving this!
Should this be possible with `max_redirects: 0`?

Reply:
ahah yes, copy/paste mistake :whistle: